.gitignore

```diff
@@ -0,0 +1,7 @@
+*-debug.log
+*-error.log
+/.nyc_output
+/dist
+/tmp
+/yarn.lock
```

Comment:
I think we should not be ignoring lock files, they are very useful for ensuring we all have the same dependency versions installed :)

Reply:
I think `oclif` generated this one automatically, coul
```suggestion
```

Reply:
you can commit auto suggestions! ☝️